addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -182,4 +185,16 @@ public void sessionModeChanged(Mode mode) {
             // Ignore
         }
     }
+
+    public void addReportedZestStatement(JSONObject json) throws Exception {
```

Comment:
Should not be public.

Reply:
This method is being used by clientIntegrationApi. Should I still not make it public?

Reply:
It does not have to and IMO this is an implementation detail not intended to be called by other add-ons. (Being package accessible is enough.)